lib/dom.js

```diff
@@ -54,6 +54,10 @@ declare class FileList {
 
 /* DOM */
 
+declare class DOMString extends string {
+
+}
+
```

Comment:
Not sure wether the following is supported:

```
declare class DOMString: string
```

If it is, it might be more appropriare given that it is a direct mapping.